lib/tasks/demo.rake

```diff
@@ -139,7 +141,6 @@ namespace :demo do
           next unless patient.created_at <= today
           next if patient.assessments.any? { |a| a.created_at.to_date == today }
           if rand < 0.9 # 70% reporting rate on any given day
-            reported_condition = patient.jurisdiction.hierarchical_condition_unpopulated_symptoms
```

Comment:
each assessment needs its own reported condition, we can't have all assessments sharing the same reported condition like this. This breaks the ability to edit the demo patient's assessments. Put this line back in and remove the line where reported condition was being made  https://github.com/SaraAlert/SaraAlert/pull/47/files#diff-0ee58a27b389079fc421e086e2c34140R129